### ISSUE TITLE
The profits are the same. Always

### ISSUE THREAD

**Issue by r-neri** [bwentzloff/trading-bot#34]
Hello guys. When I run the bot in live mode without  uncomment the lines that put the orders I observed that despite of the correct pricing values are correctly been retraived from poloniex the values of profits are are Always the same. Anymore else confirms this ?

**Comment by r-neri** [comment]
here are the output from live bot. Note the same profits.
2018-01-22 18:19:49.461692 Open: 0.028599 Close: None High: 0.02870297 Low: 0.028599 Current: 0.02870297
2018-01-22 18:20:55.084647 Open: 0.028599 Close: 0.028703 High: 0.028703 Low: 0.028599 Current: 0.028703
Price: 0.028668333333333334	Moving Average: 0.02877820407407407
Trade closed
Entry Price: 0.028796666666666665 Status: CLOSED Exit Price: 0.028668333333333334 Profit: -0.000128333333333331
2018-01-22 18:21:55.474390 Open: 0.02867797 Close: None High: 0.02867797 Low: 0.02867797 Current: 0.02867797
2018-01-22 18:22:56.921996 Open: 0.02867797 Close: None High: 0.02867797 Low: 0.02859916 Current: 0.02859916
2018-01-22 18:23:58.646495 Open: 0.02867797 Close: None High: 0.02867797 Low: 0.02859916 Current: 0.02860403
2018-01-22 18:25:03.108442 Open: 0.02867797 Close: None High: 0.02867797 Low: 0.028599 Current: 0.028599
2018-01-22 18:26:07.120509 Open: 0.02867797 Close: 0.028599 High: 0.02867797 Low: 0.028599 Current: 0.028599
Price: 0.02862532333333333	Moving Average: 0.028758796296296296
Trade opened
Entry Price: 0.028796666666666665 Status: CLOSED Exit Price: 0.028668333333333334 Profit: -0.000128333333333331
Entry Price: 0.02862532333333333 Status: OPEN Exit Price: 
2018-01-22 18:27:09.007928 Open: 0.02867692 Close: None High: 0.02867692 Low: 0.02867692 Current: 0.02867692
2018-01-22 18:28:10.945240 Open: 0.02867692 Close: None High: 0.028677 Low: 0.02867692 Current: 0.028677
2018-01-22 18:29:12.883470 Open: 0.02867692 Close: None High: 0.028677 Low: 0.02867689 Current: 0.02867689
2018-01-22 18:30:15.228001 Open: 0.02867692 Close: None High: 0.028677 Low: 0.02859901 Current: 0.02859901
2018-01-22 18:31:15.750151 Open: 0.02867692 Close: 0.028599 High: 0.028677 Low: 0.028599 Current: 0.028599
Price: 0.028625	Moving Average: 0.028739351851851853
Entry Price: 0.028796666666666665 Status: CLOSED Exit Price: 0.028668333333333334 Profit: -0.000128333333333331
Entry Price: 0.02862532333333333 Status: OPEN Exit Price: 
2018-01-22 18:32:19.244148 Open: 0.028599 Close: None High: 0.028599 Low: 0.028599 Current: 0.028599
2018-01-22 18:33:20.755611 Open: 0.028599 Close: None High: 0.02859907 Low: 0.028599 Current: 0.02859907
2018-01-22 18:34:22.368274 Open: 0.028599 Close: None High: 0.02867694 Low: 0.028599 Current: 0.02867694
2018-01-22 18:35:23.289162 Open: 0.028599 Close: None High: 0.02867694 Low: 0.028599 Current: 0.02859922
2018-01-22 18:36:24.330915 Open: 0.028599 Close: 0.028685 High: 0.028685 Low: 0.028599 Current: 0.028685
Price: 0.028656333333333336	Moving Average: 0.028723388888888885
Entry Price: 0.028796666666666665 Status: CLOSED Exit Price: 0.028668333333333334 Profit: -0.000128333333333331
Entry Price: 0.02862532333333333 Status: OPEN Exit Price: 
2018-01-22 18:37:26.380019 Open: 0.02868497 Close: None High: 0.02868497 Low: 0.02868497 Current: 0.02868497
2018-01-22 18:38:26.980810 Open: 0.02868497 Close: None High: 0.02868497 Low: 0.02860078 Current: 0.02860078
2018-01-22 18:39:27.519754 Open: 0.02868497 Close: None High: 0.02868497 Low: 0.02860078 Current: 0.02868
2018-01-22 18:40:28.810808 Open: 0.02868497 Close: None High: 0.02879409 Low: 0.02860078 Current: 0.02879409
2018-01-22 18:41:31.932180 Open: 0.02868497 Close: 0.0286804 High: 0.02879409 Low: 0.02860078 Current: 0.0286804

Any guess????
Thank you

**Comment by bwentzloff** [comment]
It’s because only one trade has closed at that point. It only tracks profit once a trade has closed.

**Comment by r-neri** [comment]
But there are more then one exit lists trade open.

**Comment by bwentzloff** [comment]
No, there’s not. Those are each a summary of all past trades. There are only two. One open and one closed.

**Comment by r-neri** [comment]
okay !  Now I Understand. Thanks.
